Assets/SEE/Layout/NodeLayouts/IncrementalTreeMap/Segment.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace SEE.Layout.NodeLayouts.IncrementalTreeMap
+{
+    /// <summary>
+    /// A segment is a core element in the layout,
+    /// it represent the line that separates the rectangles of <see cref="Node"/>s of a layout.
+    ///  The specific position is actually not relevant but the relation to adjacent nodes.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    internal class Segment
+    {
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="isConst"><see cref="IsConst"/></param>
+        /// <param name="isVertical"><see cref="IsVertical"/></param>
+        public Segment(bool isConst, bool isVertical)
+        {
+            this.IsConst = isConst;
+            this.IsVertical = isVertical;
+            this.Side1Nodes = new List<Node>();
+            this.Side2Nodes = new List<Node>();
+        }
+
+        /// <summary>
+        /// Is true if the segment is a border of the layout.
+        /// Means that <see cref="Side1Nodes"/> or <see cref="Side2Nodes"/> are empty.
```

Comment:
Actually not, because there are some rare cases in which this does not work. (in the slicing part of CorrectAreas.cs).
At least i have changed the documentation that incorrectly stated this.